player/human_player.py

```diff
@@ -0,0 +1,14 @@
+from player import Player
+
+
+class HumanPlayer(Player):
+    def choose_to_stay(self):
+        response = input('Enter \'S\' to stay, \'H\' to hit: ').lower()
```

Comment:
We can make a helper that deals with human input. To start, it would take a `string` and a `set` or `list`. Something like

```python
def get_input_from_user(self, input_string, valid_options):
    error_string = f'INVALID OPTION\n {input_string}'
    while True:
        response = input(input_string).lower()
        if response in valid_options:
             return respose
        input_string = error_string
```

Then below you could do:

```python
input_string = "Enter 'S' to stay, 'H' to hit: "
valid_options = ['s', 'h']
return get_input_from_user(input_string, valid_options) == 's'
```